packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -105,7 +104,7 @@ const VirtualListBaseFactory = (type) => {
 					scrollerNode.removeEventListener('keyup', onKeyUp, {capture: true});
 				}
 
-				pause.resume();
+				variables.current.pause.resume();
```

Comment:
The following part should be also fixed.

308 line: pause.pause();
452 line: pause.resume();

Reply:
Ah, So sorry.